docs/store/README.md

```diff
@@ -79,24 +79,24 @@ import { Observable } from 'rxjs/Observable';
 import { INCREMENT, DECREMENT, RESET } from './counter';
 
 interface AppState {
-  counter: number;
+  count: number;
 }
 
 @Component({
   selector: 'my-app',
   template: `
     <button (click)="increment()">Increment</button>
-    <div>Current Count: {{ counter | async }}</div>
+    <div>Current Count: {{ displayCount | async }}</div>
     <button (click)="decrement()">Decrement</button>
 
     <button (click)="reset()">Reset Counter</button>
   `
 })
 export class MyAppComponent {
-  counter: Observable<number>;
+  displayCount: Observable<number>;
 
   constructor(private store: Store<AppState>) {
-    this.counter = store.pipe(select('counter'));
+    this.displayCount = store.pipe(select('count'));
```

Comment:
Let's denote this as an Observable by convention also by changing it to `count$`